src/utils/authentication/authentication.types.ts

```diff
@@ -1,21 +1,7 @@
-import type { DbSession, SupabaseAuthError } from "embed-api";
-import type { JwtPayload } from "jwt-decode";
-import type { Provider, Session } from "@supabase/supabase-js";
+import type { SupabaseAuthError } from "embed-api";
+import type { Session } from "@supabase/supabase-js";
 import type { OAuthErrorCode } from "~utils/authentication/authentication.utils";
 
-export type SupabaseJwtSessionData = Omit<DbSession, "id" | "userId">;
```

Comment:
These types have been moved to the backend and are now available through `embed-api`.